src/com/facebook/buck/parser/UnexpectedFlavorException.java

```diff
@@ -114,13 +116,24 @@ public static UnexpectedFlavorException createWithSuggestions(
     return builder.build();
   }
 
+  private static ImmutableSet<Flavor> getValidFlavors(Flavored flavored, BuildTarget target) {
```

Comment:
looks like this can be replaced with a stream filter operation

Reply:
I just wanted to keep consistency across the project - other methods in the same file are using plain-old `for` loops. But that doesn't mean it's not worth refactoring, right? ;)